pkg/controller/operatorsource/operatorsourcegenerator.go

```diff
@@ -0,0 +1,18 @@
+package operatorsource
+
+import (
+	marketplacev1alpha1 "github.com/operator-framework/operator-marketplace/pkg/apis/marketplace/v1alpha1"
+)
+
+///OperatorSourceGenerator is an interface created for the purpose of mocking OperatorSources created in the operator source controller reconciliation function
```

Comment:
Extra `/`. Please try to stay within 80 column for comments.

This reminds me that we need to have `CODING.md` file.

Reply:
Updated